docs/lib/datastore/fragments/native_common/getting-started.md

```diff
@@ -72,22 +72,17 @@ Like the initial setup, models can be generated either using the IDE integration
 
 ### Code generation: Amplify CLI
 
-Models can also be generated using the Amplify CLI directly.
-
-In your terminal, make sure you are in your project/root folder and **execute the codegen command**:
-
-```console
-amplify codegen models
-```
-    
-You can **find the generated files** at `amplify/generated/models/`.
+<inline-fragment platform="js" src="~/lib/datastore/fragments/native_common/codegen.md"></inline-fragment>
+<inline-fragment platform="ios" src="~/lib/datastore/fragments/native_common/codegen.md"></inline-fragment>
+<inline-fragment platform="android" src="~/lib/datastore/fragments/native_common/codegen.md"></inline-fragment>
+<inline-fragment platform="flutter" src="~/lib/datastore/fragments/flutter/getting-started/50_codegenCli.md"></inline-fragment>
```

Comment:
Rather than doing this, what do you think about just extracting the [callout portion](https://github.com/aws-amplify/docs/pull/3275/files#diff-b2230ed1f93ab701102f38457e73c9c622aa6018d64a97a2417d5ffa9c9a47e5R12-R16) and have it be a flutter-only inline-fragment here?

Think if we have it as a one-off fragment, we wouldn't need to pull these numbered files out of sync with their platform equivalents either.